BUILD_DETAIL.md

```diff
@@ -99,15 +99,15 @@ $ bin/yard doc --no-cache
 
 ## Rubocop
 
-We use [Rubocop](https://github.com/rubocop-hq/rubocop) to enforce style conventions on the project so
+We use [Rubocop](https://github.com/bbatsov/rubocop) to enforce style conventions on the project so
```

Comment:
Regression. Please revert everything except for `FUNDING.yml`

Reply:
yep :)